app/screens/Explore.js

```diff
@@ -186,10 +191,10 @@ class Explore extends React.Component {
     console.log('onDidFailLoadingMap', err)
   }
 
-  onRegionIsChanging = async evt => {
-    // update the redux state with the bbox
-    this.props.updateVisibleBounds(await this.mapRef.getVisibleBounds(), await this.mapRef.getZoom())
-  }
+  // onRegionIsChanging = async evt => {
```

Comment:
@geohacker I think we can probably remove these commented out lines

Reply:
Let's leave it there, I think there's some part of me unsure about the map data fetch behaviour.